lib/ExpensiMark.js

```diff
@@ -476,9 +470,28 @@ export default class ExpensiMark {
             }
             replacedText = replacedText.concat(textToCheck.substr(startIndex, (match.index - startIndex)));
 
+            // we want to avoid matching email address domains
```

Comment:
```suggestion
            // We want to avoid matching email address domains
```
This comment should also say "why"

Reply:
it's from old PR, I just moved it to other line. but new comment is more readable.